developer-docs-site/docs/tutorials/build-e2e-dapp/6-handle-tasks.md

```diff
@@ -429,6 +429,8 @@ We make sure there is an account connected, set the transaction in progress stat
 ...
 ```
 
-Try it! Check a task’s checkbox, approve the transaction and see the task marked as completed.
+4. You might be getting an error for missing components. To resolve, update the import, ```import { Layout, Row, Col, Button, Spin } from "antd";``` to ```import { Layout, Row, Col, Button, Spin, List, Checkbox, Input } from "antd";```.
```

Comment:
Thanks! We probably need to ask to update the import statement on the step we use the new components